bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/TypeChecker.java

```diff
@@ -551,6 +551,11 @@ public static Object getAnnotValue(TypedescValue typedescValue, BString annotTag
         if (!(describingType instanceof BAnnotatableType annotatableType)) {
             return null;
         }
+        MapValue annotations =  ((TypedescValueImpl) typedescValue).annotations;
```

Comment:
Why do we need this change? Are these annotations only in the typedesc, not in the type?

Reply:
They are in both the places. I think we can remove this for now

Reply:
Removed this